retirement/tests/tests_viewset_Reservation.py

```diff
@@ -52,6 +52,7 @@ class ReservationTests(APITestCase):
     @classmethod
     def setUpClass(cls):
         super(ReservationTests, cls).setUpClass()
+        cls.maxDiff=None
```

Comment:
Please remove this since it's just for your local test

Reply:
done